src/haskell/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -e
+
+BOOTSTRAP_HASKELL_GHC_VERSION="${VERSION:-"recommended "}"
+
+# Maybe install curl, gcc, make
+for x in curl gcc make; do
+    which $x > /dev/null || (apt update && apt install $x -y -qq)
+done
+
+curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | BOOTSTRAP_HASKELL_NONINTERACTIVE=1 BOOTSTRAP_HASKELL_MINIMAL=1 sh
```

Comment:
Downloading files and executing them makes it prone to supply chain attacks. We have https://github.com/devcontainers/features/issues/412 opened which fixes current occurrences. 

Can we verify integrity of this file?

Reply:
switched to direct download and sha verify.